.github/workflows/manylinux.yml

```diff
@@ -33,12 +33,14 @@ jobs:
       fail-fast: false  # if a particular matrix build fails, don't skip the rest
       matrix:
         arch: ['x64', 'x86', 'aarch64']
-        tag: ['-', '-manylinux-'] # one tag for CPython builds, and another for PyPy
+        tag: ['-manylinux1-', '-manylinux2010-', -manylinux2014-]
```

Comment:
minor nitpick here, maybe the leading and trailing hyphen `-` must not be in the `matrix.tag` rather it can come a few lines below, where `make pull${{ matrix.tag }}${{ matrix.arch }} wheels${{ matrix.tag }}${{ matrix.arch }}` should become `make pull-${{ matrix.tag }}-${{ matrix.arch }} wheels-${{ matrix.tag }-}${{ matrix.arch }}`

This is so that the build name looks neater :)

Reply:
Good point. I'll fix that after the release :)